### ISSUE TITLE
Proton stopped working completly

### ISSUE THREAD

**Issue by gmBlahaj** [ValveSoftware/Proton#7125]
Today for some reason proton completly stopped working, every single version. Here is a sample Log file
[steam-1118200.log](https://github.com/ValveSoftware/Proton/files/12707320/steam-1118200.log)

**Comment by gmBlahaj** [comment]
Some extra info:
OS: Arch Linux
GPU: Nvidia RTX 3070 TI (nvidia drivers)
RAM: 32GB
Kernel: 6.5.4-arch2-1

**Comment by AlexBarrett12** [comment]
Looks to me like line 1617 is causing the issue: 
```
242.415:0041:0042:err:module:import_dll Loading library OPENGL32.dll (which is needed by L"E:\\SteamLibrary\\steamapps\\common\\People Playground\\UnityPlayer.dll") failed (error c000007b).
```
0xc000007b error is apparently caused by trying to execute a 32-bit binary from a 64-bit binary. Basically, seems like something is wrong with this particular dll file.

Briefly testing this (same game, same proton version), seems like this is the expected log for loading opengl32.dll:
```
54467.075:002f:0030:trace:loaddll:load_so_dll Loaded L"C:\\windows\\system32\\OPENGL32.dll" at 0x7a850000: builtin
```
which is actually loading the dll from an so file under 'steamapps/common/Proton 4.11/dist/lib64/wine/opengl32.dll.so' (when running 4.11 natively in Steam via the compatibility option 'Force the use of a specific Steam Play compatibility tool')

I could recreate a similar error by deleting this file (0x4000000e error instead, but everything else is the same). Notably, running the 'verify integrity' check for that version of proton did not repair this. Neither did uninstalling and reinstalling that version of proton via Steam.

Seems like under 'steamapps/common/Proton [version]' there is a `proton_dist.tar` that gets extracted into the `dist` folder (right next to it in the same folder). But, this only happens once and never again even if you uninstall and reinstall this version of proton (the files remain even when 'uninstalled').

Basically, either try manually extracting `proton_dist.tar` to `dist` via:
```
tar -xf proton_dist.tar -C ./dist/
```
Or, uninstall the Proton version in Steam, then head to 'steamapps/common' and delete the 'Proton [version]' folder that you just uninstalled. Reinstalling should restore the files.

Either that or rebuild from source like a legend :)

Let me know if that works or if we start unlocking new errors.

**Comment by kisak-valve** [comment]
If you suspect that a Proton install is damaged or incomplete, you can delete the `dist` folder, and Proton will regenerate that folder the next time it is run.

**Comment by jpx40** [comment]
I experience similar issues to proton, it is unusable for me at the moment.
